f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -521,7 +529,7 @@ def _create(self, bigip, vs, listener, service):
         self.driver.service_adapter.get_vlan(vs, bigip, network_id)
         if listener['protocol'] == "HTTP" or \
            listener['protocol'] == "TERMINATED_HTTPS":
-            self._create_http_profile(bigip, vs)
+            self._create_http_profile(bigip, listener, vs)
```

Comment:
I am not 100% sure if "customized" json can be passed by listener dict. That depends on driver code. Maybe it can be passed by service['listener']. We need to check that during testing.

Reply:
> I am not 100% sure if "customized" json can be passed by listener dict. That depends on driver code. Maybe it can be passed by service['listener']. We need to check that during testing.

I tested the code, listener['customized'] works.